functions/src/models/monitor.model.ts

```diff
@@ -11,4 +11,6 @@ export class MonitorModel {
   path: string;
   uid: string;
   latestPing: PingModel;
+  successPingCounts: number;
```

Comment:
Can this be renamed to `successfulPings` and `unsuccessfulPings`

Reply:
Sure